index.js

```diff
@@ -10,8 +10,13 @@ const OK = chalk.green('Downloaded:');
 const FROM = chalk.gray('from');
 
 function download(repo, logger) {
-    utils.downloadRepo(repo).then((files) => {
-        files.forEach(file => logger.info(OK, `${file} ${FROM} ${repo}`));
+    utils.downloadRepo(repo).then(({ files, dependencies }) => {
+        if (files && files.length) {
+            files.forEach(file => logger.info(OK, `${file} ${FROM} ${repo}`));
+        }
+        if (dependencies && dependencies.length) {
+            dependencies.forEach(dep => logger.info(OK, `${dep} ${FROM} ${repo}`));
```

Comment:
Also, if all dependencies/files tunnel through the same handler, it'd be easier to implement the `Overwrite?` prompt because it'd be a single place.